### PR TITLE
Adds GHA to verify records are written into Hasura via the Faros destination

### LINKED ISSUE #95: Automated integration tests
We currently have a GitHub Action ([ci-cd](https://github.com/faros-ai/faros-community-edition/blob/main/.github/workflows/ci-cd.yml)) start a local Faros CE with docker-compose and runs automated tests. Those tests are currently testing services in isolation. We want to augment the scope of those test to capture how those services interact. 

In particular:
* Airbyte Faros Destination pushing data into the Faros DB through the Hasura GraphQL endpoint
* Metabase being able to read that data

### PR DESCRIPTION
**Pull request by ypc-faros**
# Description

This change ensures that:

- There are no issues with the Hasura url with which the Faros destination is configured (i.e., Airbyte can connect to Hasura)
- When records are written via the Faros destination, these are persisted in Hasura.

Notes:
We use the destination id from [the Airbyte destination cfg](https://github.com/faros-ai/faros-community-edition/blob/main/init/resources/airbyte/workspace/airbyte_config/DESTINATION_CONNECTION.yaml)
We need to also read the Hasura admin secret from the [.env](https://github.com/faros-ai/faros-community-edition/blob/main/.env) file.
We expect the "live" destination id and Hasura admin secret to match the values in these files. So if there are failures because of a mismatch we might have uncovered a bug in the (Airbyte and/or Hasura) initialization.

Fixes #95 

## Type of change
- New feature (non-breaking change which adds functionality)

# Checklist
- [X] Have you checked to there aren't other open Pull Requests for the same update/change?
- [X] Have you added an explanation of what your changes do and why you'd like us to include them?

### CODE REVIEW

File: .github/workflows/resources/faros-destination/catalog.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
+{
```
Comment (tovbinm) [inline .github/workflows/resources/faros-destination/catalog.json:1]: Lets avoid storing resources in `.github` folder. This should be only used for repo management files, e.g. templates, workflows.

Comment (tovbinm) [reply .github/workflows/resources/faros-destination/catalog.json:1]: I would move them to test folder (see my earlier comment - https://github.com/faros-ai/faros-community-edition/pull/98/files#r845318885)

File: .github/workflows/ci-cd.yml [hunk 2]
```diff
@@ -54,6 +53,53 @@ jobs:
       - name: Show services
         run: docker-compose ps -a
 
+      - name: Get destination id
```
[the remark this answers is not in the document]
Comment (ypc-faros) [reply .github/workflows/ci-cd.yml:56]: Do you mean the whole thing or some steps in particular?

I found it convenient to use `jq` and `yq` to extract the destination id from the yaml file and inject the Hasura admin secret in the destination configuration returned by Airbyte (the Hasura admin secret is redacted in the response).

I'm sure all of these can be accomplished in TypeScript, it just felt simpler to implement like this for me (as a noob to both Shell scripting and TypeScript :) )

**Comment by ypc-faros** [comment]
Closed in favor of #102

**Pull request closed without merging**